Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI_test.py

```diff
@@ -0,0 +1,74 @@
+import json
+import os
+
+import pytest
+
+from MicrosoftGraphAPI import MsGraphClient, generic_command
+
+
+def load_test_data(test_data_filename):
+    with open(os.path.join('./test_data', test_data_filename)) as f:
+        return json.load(f)
+
+
+@pytest.fixture()
+def client(requests_mock):
+    requests_mock.post(
+        'https://login.microsoftonline.com/tenant_id/oauth2/v2.0/token',
+        json={},
+    )
+    return MsGraphClient(
+        app_id='app_id',
+        scope='Application.Read.All',
+        app_secret='app_secret',
+        tenant_id='tenant_id',
+        verify=False,
+        proxy=False,
+    )
+
+
+def test_generic_command_positive(requests_mock, client):
+    """
+    Given:
+        - API resource /applications
+    When:
+        - Running the generic command
+    Then:
+        - Ensure outputs are as expected
+    """
+    applications_res = load_test_data('applications.json')
+    requests_mock.get(
+        'https://graph.microsoft.com/v1.0/applications?$top=10',
+        json=applications_res,
+    )
+    args = {
+        'resource': '/applications',
+        'odata': '$top=10',
```

Comment:
since you removed the default value, what will get returned if you drop this? how many entries?

Reply:
as many as there are
it was the concern initially and thats why i set it as default, but it breaks some apis and causes bad ux
got another idea ?

Reply:
maybe state it in the command/argument description?

Reply:
done, can you have a look please?